cpp/common/coverage/collect_coverage.c

```diff
@@ -0,0 +1,27 @@
+
+#if defined(TOSCA_COVERAGE)
+extern void __gcov_dump();
+#endif
+
+/// Reports whenever the library (and therefore the complete c++ project)
```

Comment:
Just a question: what's with the three `///` here instead of the usual two? Does this have any particular meaning?

Reply:
It is one of the methods to document functions using doxygen. https://www.doxygen.nl/manual/docblocks.html
Do we use some particular notation for documentation comments?  I have noticed `// <` in some places.

Reply:
this was mostly muscle memory, shall we use something else?

Reply:
Seeing a `///` instead of a `//` makes me think that there is some special meaning for it, which, based on your feedback, there is not. So it should be rewritten to use `//`.

About the `// <` I use some times: I do this for fields, if I do not want to break the formatting.

When you have a struct like
```
type myData struct {
   short                 T
   longFieldName S
}
```
Go-format is nicely aligning the types. However, when adding documentation between those fields, this is lost:
```
type myData struct {
   // short is ...
   short T
   // longFieldName is ...
   longFieldName S
}
```
and it quickly looks messy. Thus, I use end-of-line comments like this:
```
type myData struct {
   short                 T  // < this is ..
   longFieldName S  // < this is ...
}
```
and the formatter makes it look good. Here, the `<` is to point to the thing it is documenting.

I saw this supported once in some doc generation tool, but I never checked what Go would make of it. I see those comments to be consumed mainly in-code.

Reply:
Well, there is a meaning: // is a comment, /// is documentation. Any tool processing documentation will ignore // comments. I see no use for this now, I'll remove them.